websites/E/e-z/presence.ts

```diff
@@ -0,0 +1,115 @@
+const presence = new Presence({
+	clientId: "1018191690210754693",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://cdn.e-z.host/e-zimagehosting/ez256.png",
+		},
+		username = document.querySelector("p[id=premid-username]")?.textContent,
+		uid = document.querySelector("p[id=premid-uid]")?.textContent;
+
+	switch (document.location.hostname) {
```

Comment:
Use variables for this.
[Example of the right variables.](https://github.com/PreMiD/Presences/commit/958d5b80733887427f412471048b9614390e1fb7#diff-adb2cc91e493add5ee896439dcca4f8073c1cda9b686816134ca8910eba6c8acR24)